### PR TITLE
fix: multiline output

### PR DESCRIPTION
**Pull request by jedevc**
Fixes https://github.com/dagger/dagger-for-github/issues/161, a regression introduced in [`1a00b8f` (#153)](https://github.com/dagger/dagger-for-github/pull/153/commits/1a00b8f5596e2b3e59f2c7d3e47953dd6c4de7b7).

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -83,7 +83,16 @@ runs:
...
-        (echo -n "stdout=" && cat "${tmpout}") >> "$GITHUB_OUTPUT"
+        {
+          # we need a delim that doesn't appear in the output - a hash of the
+          # file itself *probably* won't (if it does, we have larger
+          # cryptographic problems)
+          delim="$(sha256sum $tmpout | cut -d " " -f1)"
+          echo "stdout<<${delim}"
+          cat "${tmpout}"
+          echo
+          echo "${delim}"
```
Comment (kpenfound) [inline action.yml:94]: That is very creative

**Comment by jedevc** [comment]
FYI @kpenfound added a commit at the end that fixes https://github.com/dagger/dagger-for-github/pull/162.

Noticed there was a bug that caused quotes in `args`/`call` to not *quite* work as expected, they were getting incorrectly expanded. So just fell back to the old behavior, and handled the replacement in github action expressions.

File: action.yml (hunk 1 above)
Comment (jedevc) [reply action.yml:99]: :stuck_out_tongue_winking_eye: hehe ye, better suggestions appreciated, I didn't want to use a fixed `EOF` string on the off chance that that appeared in the output.

**Review by kpenfound — APPROVED** [review]
ship it

**Pull request merged**